pkg/geth/node/node.go

```diff
@@ -5,45 +5,82 @@ import (
 
 	"strconv"
 
+	"regexp"
+
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/p2p"
 	"github.com/ethereum/go-ethereum/rpc"
 	"github.com/vulcanize/vulcanizedb/pkg/core"
 )
 
-func Info(client *rpc.Client) core.Node {
+type Getter interface {
```

Comment:
Wondering if there's a better name for this interface - feels more specific than a `Getter`